### ISSUE TITLE
Publish RPM for SUSE

### ISSUE THREAD

**Issue by brunoborges** [microsoft/openjdk#389]
- [ ] Publish RPM on packages.microsoft.com - @d3r3kk 
- [ ] Update documentation - @brunoborges

**Comment by soloturn** [comment]
suse already has openjdk, @brunoborges :
https://software.opensuse.org/search?baseproject=ALL&q=openjdk

**Comment by karianna** [comment]
@soloturn FYI Temurin supports Suse installers.  If it's not appearing in their official index that's a separate matter, please do report it over there!

**Comment by soloturn** [comment]
this can be closed i guess @karianna ?

**Comment by karianna** [comment]
We'll keep this open for the Microsoft Build of OpenJDK consideration

**Comment by AdityaMayukhSom** [comment]
Any updates on publishing Microsoft build of OpenJDK to openSUSE?

**Comment by karianna** [comment]
> Any updates on publishing Microsoft build of OpenJDK to openSUSE?

Would love to hear aboit your use case.  Are you using openSUSE on Azure?

**Comment by AdityaMayukhSom** [comment]
Yes, currently using Amazon Corretto. It is not causing any specific issue per se. But we would like to have some alternative directly from microsoft prod repository as we install microsoft production repository for installing dotnet. For corretto we need to add one more repository from Amazon.

**Comment by karianna** [comment]
@AdityaMayukhSom - Can you try installing as per https://learn.microsoft.com/en-us/java/openjdk/install#install-on-suse :-)

**Comment by karianna** [comment]
@soloturn If you could also try that would be great!
